src/Workspaces/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,2 @@
-
+Microsoft.CodeAnalysis.DocumentInfo.DesignTimeOnly.get -> bool
+Microsoft.CodeAnalysis.DocumentInfo.DiagnosticsLspClientName.get -> string
```

Comment:
public API change.  so this will need more of a review. tagging @jasonmalinowski